### DISCUSSION TITLE
Map attributes should display on map card when leaderboards are showing

### DISCUSSION THREAD

**Discussion by Bubbleman532** [ppy/osu#25055] [Ideas] [answered]
The only way to see the AR/CS/OD/HP of a map currently is on the details tab. This makes it impossible to see leaderboards at the same time as the map difficulty settings, and players are used to being able to see these details at a glance.

To standard players at least, the map settings are more important information than the circle, slider and spinner counts.

While I enjoy the addition of a Details tab and probably will use it most often since I dislike seeing leaderboards at all times, most players have leaderboards as their default so would greatly appreciate the extra information being available to them without having to check the details tab each time.

**Comment by Joehuu** [comment, marked as answer]
The [new design](https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1) aims to display both. Progress is tracked in https://github.com/ppy/osu/issues/22204.
